### PR TITLE
[vs] Fix issue of swss debian being overwritten by sairedis

### PR DESCRIPTION
**Pull request by daall**
Signed-off-by: Danny Allen <daall@microsoft.com>

Fixes Azure/sonic-swss#1404

The addition of `sonic-sairedis` VS testing (https://github.com/Azure/sonic-build-tools/pull/133) led to `swss_1.0.0_amd64.deb` being added to the list of `sairedis` artifacts. Because of this change, the artifacts from `sairedis`, which is based on the latest `swss`, were overwriting the PR-built `swss` package.

This change narrows the list of artifacts being copied for the DVS build so that this does not happen again in the future. It also moves the swss install to the end.

### CODE REVIEW

File: jenkins/vs/sonic-swss-build-pr/Jenkinsfile [hunk 1]
```diff
@@ -10,10 +10,10 @@ pipeline {
                               userRemoteConfigs: [[url: 'https://github.com/Azure/sonic-swss',
                                                    refspec: '+refs/pull/*:refs/remotes/origin/pr/*']]])
                 }
-                copyArtifacts(projectName: 'sonic-sairedis-build', filter: '**/*.deb', target: 'sairedis', flatten: true)
+                copyArtifacts(projectName: 'vs/sonic-sairedis-build', filter: '**/*.deb', target: 'sairedis', flatten: true)
```
Comment (abdosi) [inline jenkins/vs/sonic-swss-build-pr/Jenkinsfile:13]: @daall is this change related to issue ?

Comment (daall) [reply jenkins/vs/sonic-swss-build-pr/Jenkinsfile:13]: No, I added this so I can test the change since the `jenkins_test` folder isn't in the `vs`.

**Comment by daall** [comment]
https://sonic-jenkins.westus2.cloudapp.azure.com/job/common/job/jenkins_testing/job/swss-test/34/console

```
cp libsaimetadata-dbg_1.0.0_amd64.deb libsaimetadata-dev_1.0.0_amd64.deb libsaimetadata_1.0.0_amd64.deb libsairedis-dbg_1.0.0_amd64.deb libsairedis-dev_1.0.0_amd64.deb libsairedis_1.0.0_amd64.deb libsaivs-dbg_1.0.0_amd64.deb libsaivs-dev_1.0.0_amd64.deb libsaivs_1.0.0_amd64.deb swss-dbg_1.0.0_amd64.deb swss_1.0.0_amd64.deb syncd-vs-dbg_1.0.0_amd64.deb syncd-vs_1.0.0_amd64.deb scripts/vs/sonic-sairedis-build/docker-sonic-vs/debs

cp common/libswsscommon-dbg_1.0.0_amd64.deb common/libswsscommon-dev_1.0.0_amd64.deb common/libswsscommon_1.0.0_amd64.deb common/python-swsscommon_1.0.0_amd64.deb common/python3-swsscommon_1.0.0_amd64.deb scripts/vs/sonic-sairedis-build/docker-sonic-vs/debs
```

Verified that only the necessary packages are copied, and the order is correct. Waiting on the test result to verify that the acl ERRORs are not related to this PR.

**Comment by daall** [comment]
The lone test failure was due to https://github.com/Azure/sonic-buildimage/issues/5205.
